packages/application-shell/src/configure-apollo.ts

```diff
@@ -1,34 +1,32 @@
-import type { IdGetterObj } from 'apollo-cache-inmemory';
 import type { ApplicationWindow } from '@commercetools-frontend/constants';
 
-import ApolloClient from 'apollo-client';
-import { version as apolloVersion } from 'apollo-client/version';
-import { ApolloLink } from 'apollo-link';
-import apolloLogger from 'apollo-link-logger';
-import { createHttpLink } from 'apollo-link-http';
 import {
+  ApolloClient,
+  from,
+  createHttpLink,
   InMemoryCache,
-  IntrospectionFragmentMatcher,
-} from 'apollo-cache-inmemory';
+} from '@apollo/client';
+// import * as apolloVersion from '@apollo/client/version';
```

Comment:
Apollo does not expose a `version` anymore.

Actually it does but it's in ES6 format, which breaks bundling.